lib/web_ui/dev/roll_fallback_fonts.dart

```diff
@@ -552,12 +564,15 @@ Future<bool> _checkForLicenseAttribution(
 }
 
 class _Font {
-  _Font(this.family, this.index, this.starts, this.ends);
+  _Font(this.family, this.index, this.starts, this.ends, {
+    this.uri,
+  });
 
   final String family;
   final int index;
   final List<int> starts;
   final List<int> ends; // inclusive ends
+  final Uri? uri;
```

Comment:
Why nullable?

Reply:
There was some code checking for null uris in the old code, but it comes from putting the uri in a Map, not really because the uri can be null in any normal flow.